src/ExpressiveInstaller/OptionalPackages.php

```diff
@@ -384,6 +385,20 @@ public static function copyFile(IOInterface $io, $projectRoot, $source, $target,
     }
 
     /**
+     * Remove line from content.
+     *
+     * @param string $entry Entry to remove.
+     * @param string $content String to remove entry from.
+     * @return string
+     */
+    public static function removeLine($entry, $content)
```

Comment:
It did at first, but it didn't feel right since it isn't necessarily a file that is passed to it. But I can change it back.